### ISSUE TITLE
Sidebar doesn't stay collapsed

### ISSUE THREAD

**Issue by thany** [vector-im/element-web#23087]
### Steps to reproduce

1. Collapse the sidebar
2. Make the window narrower
3. Make the window wider again

#### What did you expect?
Sidebar stays as big as it is, even if that means keeping it collapsed.

#### What happened instead?
Sidebar un-collapses.

### Operating system

Windows 10

### Application version

Element version: 1.11.3 Olm version: 3.2.12

### Will you send logs?

No

**Comment by MichaelSasser** [comment]
Yeah, that is super annoying, especially on small screens with a tiling window manager. Every time I close the developer tools (web browser), or move them to a different screen, the "room list sidebar" expands.
In general, it wouldn't bother me that much, but when I have threads open and the sidebar is expanded, I see about seven words in the main chat window with the IRC layout (and developer tools closed).

**Comment by nadonomy** [comment]
+1 to persisting the same state

**Comment by thany** [comment]
Any news on implementing that option to *never* expand/collapse it automatically?
